Cartfile.resolved

```diff
@@ -1,8 +1,8 @@
 github "LoopKit/Amplitude-iOS" "2137d5fd44bf630ed33e1e72d7af6d8f8612f270"
```

Comment:
Since the `Cartfile` and `Cartfile.resolved` reference the DIY repos (i.e. `LoopKit` prefix), this file should not be updated unless you actually pulled in changes from the DIY repo. And, if you did that then the DIY pull should be a completely separate PR explicitly for pulling in DIY changes.